src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkCorrectUsingMethodsInspection.kt

```diff
@@ -43,19 +52,21 @@ class SmkCorrectUsingMethodsInspection: SnakemakeInspection() {
                 return
             }
 
+            val firstChildSection = st.firstChild
+
             st.argumentList?.node?.getChildren(null)
```

Comment:
Could we use `call expression` filter in the line 57 instead of `null`?

Reply:
Fixed it